app/schemas.py

```diff
@@ -0,0 +1,96 @@
+from pydantic import BaseModel
+from typing import List, Optional
+from datetime import datetime
+
+
+class LibraryBase(BaseModel):
+    name: str
+    address: str
+
+
+class LibraryCreate(LibraryBase):
+    pass
+
+
+class Library(LibraryBase):
```

Comment:
We can rename the schemas that have the same name as the models to prevent import conflict. It can be LibrarySchema